### PR TITLE
EA PG DebouncerEvents - Part 1

### PR DESCRIPTION
**Pull request by oetherington**
Part 2 in #6686 

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203968439162171) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by jpaddison3** [comment]
What a PR. I'm behind, but have at least started this and hope to get to it by tomorrow morning EST. (Hopefully™️)

File: packages/lesswrong/lib/sql/InsertQuery.ts [hunk 1]
```diff
@@ -3,6 +3,12 @@ import Table from "./Table";
 import { Type } from "./Type";
```
Comment (jpaddison3) [inline packages/lesswrong/lib/sql/InsertQuery.ts:3]: This file is the only one I have left.

**Review by jpaddison3 — COMMENTED** [review]
Partial review. Let's talk live tomorrow.

File: packages/lesswrong/lib/sql/PgCollection.ts [hunk 2]
```diff
@@ -203,6 +208,7 @@ class PgCollection<T extends DbObject> extends MongoCollection<T> {
   }
 
   rawRemove = async (selector: string | MongoSelector<T>, options?: MongoRemoveOptions<T>) => {
+    options = Object.assign({noSafetyHarness: true}, options);
```
[the remark this answers is not in the document]
Comment (oetherington) [reply packages/lesswrong/lib/sql/PgCollection.ts:211]: Yeah - in the query builder interface we can offer this extra security, but Mongo doesn't have this security so adding it in now breaks backwards-compatability.

**Pull request merged**